src/app/views/datasets/DatasetsController.jsx

```diff
@@ -113,58 +108,74 @@ class DatasetsController extends Component {
             console.error("Error getting dataset recipes:\n", error);
         });
     }
+    
+    mapResponseToTableData(datasets, instances) {
+        var values = [];
+        var addedDatasets = [];
 
-    mapAPIResponsesToViewProps(completedInstances, datasets) {
-        // TODO once import API stores uploader info we want to map it to the completed dataset for the view to display
-        return completedInstances.map(instance => {
-            const instanceDataset = datasets.find(dataset => {
-                return dataset.id === instance.dataset_id
-            });
-            return {
-                id: instance.id,
-                name: instanceDataset.title || `No name available (${instance.id})`
-            }
-        });
-    }
+        instances.map(instance => {
```

Comment:
I currently get an empty list because I have 1 dataset on my local but no instances, because we loop round instances instead of datasets. The solution to this might be reworking this loop so it goes through datasets instead, but this probably needs a bit of extra thought.